common/men_at_arms_types/00_maa_types.txt

```diff
@@ -169,9 +199,69 @@ onager = {
 	
 	stack = 10
 	allowed_in_hired_troops = no
+	icon = seige_weapon
+}
+
+giant_unit = {
+	type = massive_infantry
+	
+	damage = 36
+	toughness = 35
+	pursuit = 0
+	screen = 13
+	
+	terrain_bonus = {
+		wetlands = { damage = -3 pursuit = -6 }
+		jungle = { damage = -5 pursuit = -10 }
+		mountains = { damage = -4 pursuit = -8 }
+
+	}
+
+	counters = {
+		heavy_infantry = 1
+		light_cavalry = 1
+	}
+
+	buy_cost = { gold = massive_infantry_recruitment_cost }
+	low_maintenance_cost = { gold = massive_infantry_low_maint_cost }
+	high_maintenance_cost = { gold = massive_infantry_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = culture_ai_weight_heavy_cavalry }
+	icon = massive_infantry
+
 }
 
+
 #Medieval Era
+aerial_unit = {
```

Comment:
Has no icon.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/102015443-05a37b00-3d75-11eb-9e03-b1a13bed4544.png)

</details>